### PR TITLE
Add support for server play map sound

### PR DESCRIPTION
**Pull request by Bamcane**
This is sound part of solving #7296 

The reason you can see in #8380 

## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Review by def- — APPROVED** [review]

File: datasrc/network.py [hunk 1]
```diff
@@ -585,4 +589,8 @@
 	NetMessageEx("Sv_ChangeInfoCooldown", "change-info-cooldown@netmsg.ddnet.org", [
 		NetTick("m_WaitUntil")
 	]),
+
+	NetMessageEx("Sv_MapSoundGlobal", "map-sound-global@netmsg.ddnet.org", [
+		NetIntAny("m_SoundId", 0),
```
Comment (heinrich5991) [inline datasrc/network.py:594]: Was the 0 intentional? If not:
```suggestion
		NetIntAny("m_SoundId"),
```

File: datasrc/network.py [hunk 2]
```diff
@@ -374,6 +374,10 @@
 		NetIntAny("m_Layer"),
 		NetIntAny("m_EntityClass"),
 	]),
+
+	NetEventEx("MapSoundWorld:Common", "map-sound-world@netevent.ddnet.org", [
+		NetIntAny("m_SoundId", 0),
```
Comment (heinrich5991) [inline datasrc/network.py:379]: Was the 0 intentional? If not:
```suggestion
		NetIntAny("m_SoundId"),
```

**Review by heinrich5991 — COMMENTED** [review]
Looks good.

**Comment by heinrich5991** [comment]
Can you squash the commits? Should I do it?

**Comment by Bamcane** [comment]
squash finish

**Review by heinrich5991 — APPROVED** [review]
Thanks!

**Comment by Kaffeine** [comment]
Hi. Is there a reason why `SoundGlobal`, `SoundWorld`, and `MapSoundWorld` are NetEvents, and `Sv_MapSoundGlobal` is a NetMessage? I'm new here.

**Pull request merged**